tests/decorators/test_validation.py

```diff
@@ -43,6 +43,13 @@ def test_get_nullable_parameter():
     assert result is None
 
 
+def test_get_explodable_object_parameter():
```

Comment:
what does a non-exploded deepObject look like?
it's unclean from the spec, but do you support more than one level deep like foo[bar][baz]=5?

Reply:
> what does a non-exploded deepObject look like?

according to the [spec](https://swagger.io/docs/specification/serialization/#query) thats not possible and not a thing. not sure how to handle this.

>  but do you support more than one level deep like foo[bar][baz]=5?

will check on the possibility to include that ...

Reply:
> will check on the possibility to include that ...

added